solr/solr-ref-guide/modules/deployment-guide/pages/zookeeper-access-control.adoc

```diff
@@ -56,9 +56,37 @@ ACLs describe who is allowed to read, update, delete, create, etc.
 Each piece of information (znode/content) in ZooKeeper has its own set of ACLs, and inheritance or sharing is not possible.
 The default behavior in Solr is to add one ACL on all the content it creates - one ACL that gives anyone the permission to do anything (in ZooKeeper terms this is called "the open-unsafe ACL").
 
+
+
+== Solr to Zookeeper ACLs Workflow
+
+* Solr to Zookeeper credentials and ACLs are controlled through 3 interfaces: {solr-javadocs}/solrj/org/apache/solr/common/cloud/ZkCredentialsInjector.html[`ZkCredentialsInjector`],  {solr-javadocs}/solrj/org/apache/solr/common/cloud/ZkCredentialsProvider.html[`ZkCredentialsProvider`] and {solr-javadocs}/solrj/org/apache/solr/common/cloud/ZkACLProvider.html[`ZkACLProvider`].
+
+* The classes implementing these 3 interfaces are passed to Solr via System Properties using the following properties names:
+`zkCredentialsInjector`, `zkACLProvider` and `zkCredentialsProvider`. See below sections for details.
```

Comment:
I wonder if it makes sense to display this information in a tabular form.

Class | System Property | Description
---|---|---
ZkCredentialsInjector | zkCredentialsInjector | Class which gets credentials from some source
ZkCredentialsProvider | ... | ...

Also, the system properties are defined by the contents of `solr.xml` they are not necessarily universal. We should have ref to `configuring-solr-xml.adoc` here for that.

Reply:
Added a ref to configuring-solr-xml.adoc and changed the wording little bit. Will add a tubular later on.